modules/core/src/main/scala/org/tessellation/infrastructure/trust/storage/TrustStorage.scala

```diff
@@ -78,11 +89,37 @@ object TrustStorage {
 
       def getTrust: F[TrustMap] = trustStoreRef.get.map(_.trust)
 
+      def getBiasedTrust: F[TrustMap] = getTrust.map { trust =>
+        seedlist.fold(trust) { peerIds =>
+          val updatedTrust = peerIds
+            .filterNot(trust.trust.contains)
+            .map(_ -> TrustInfo(predictedTrust = config.seedlistInputBias.some))
```

Comment:
`map(_ => a)` == `.as(a)`

Reply:
@marcinwadon Looking into it I don't think we can do that, unless you know the correct import? `peerIds` is a `Set` which I think is not a _Functor_ so that import doesn't help us. Even `import cats.syntax.all._` doesn't resolve it.

Reply:
ok, didn't see it's not on the `Functor`. you're right :+1: